### PR TITLE
Added team views

### PR DESCRIPTION
**Pull request by makarman**
Added team urls to eventually urls
Added team views

## JIRA

* [Main JIRA ticket](http://ssu-jira.softserveinc.com/browse/LVPYTP-35)
## Code reviewers

- [ ] @dmytrorevak
- [ ] @IvanAndrukh
- [ ] @JjuliaP
- [ ] @makarman
- [ ] @noobhunters
- [ ] @Silent777
- [ ] @Staryj812

### Second Level Review

- [ ] @ssOleg
- [ ] @lhalam

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: eventually/team/models.py [hunk 1]
```diff
@@ -150,20 +150,25 @@ def create(owner, members, name=None, description='', image=''):
 
         :return: Team object instance or None if object does not exist
         """
-
-        team = Team()
-        team.name = name
-        team.description = description
-        team.image = image
-        team.owner = owner
+        data = {}
```
Comment (redmyt) [inline eventually/team/models.py]: for what we add this `data` dict?

File: eventually/team/models.py [hunk 2]
```diff
@@ -181,13 +186,14 @@ def update(self, owner=None, members=None, name=None, description=None, image=No
...
         :param image: team's image
         :type image: str
-
         :return: None
         """
         if owner:
             self.owner = owner
-        if members:
-            self.members = members
+        if members_add:
```
Comment (redmyt) [inline eventually/team/models.py:203]: what if we would create additional functions `add_users` and `remove_users` like the model's methods?
what do you think?

File: eventually/team/models.py (hunk 1 above)
Comment (lhalam) [reply eventually/team/models.py]: roll back

File: eventually/team/views.py [hunk 3]
```diff
@@ -3,8 +3,130 @@
...
+        Handles GET request, retrieves team from the database
+        :param request: request from the web page
+        :type request: http Request
+
+        :param team_id: id of a team to return.
+        :type team_id: int
+
+        :return: team dictionary
+            E.G.
+                {
```
Comment (lhalam) [inline eventually/team/views.py]: add `|`

File: eventually/team/views.py [hunk 4]
```diff
@@ -3,8 +3,130 @@
...
+                    ]
+                }
+        """
+        if team_id:
+            team = Team.get_by_id(team_id)
+            if team:
+                team = team.to_dict()
+                return JsonResponse(team, status=200)
+            return HttpResponse(status=404)
+        else:
```
Comment (lhalam) [inline eventually/team/views.py]: unnecessary `else`

**Review by lhalam — CHANGES_REQUESTED** [review]

**Review by lhalam — APPROVED** [review]

**Pull request merged**